### PR TITLE
Modularize related

### PR DESCRIPTION
**Pull request by Chrisgood2go**
got rid of error when click star

### CODE REVIEW

File: client/src/components/relatedProducts/card.jsx [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 import styled from 'styled-components';
 import TestStarIcon from '../stars/star.jsx'
+//////////
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import {faStar} from '@fortawesome/free-solid-svg-icons'
+
+let second= {
+  color: 'cornflowerblue',
+}
```
Comment (Nkozlare) [inline client/src/components/relatedProducts/card.jsx:12]: You can import the different icons on the same line, like 
import { faCircleXmark, faStar } from '@fortawesome/free-solid-svg-icons'

File: client/src/components/relatedProducts/card.jsx [hunk 2]
```diff
@@ -106,12 +114,19 @@ const Card = (props) => {
...
   return (
     <Carta >
-      <RelatedBtn>{props.button}</RelatedBtn>
+      <RelatedBtn  onClick={()=>{
+        console.log(`clicked ${props.btnStyle} button`)
+        props.btnStyle === 'x' ? props.handleXClick() : props.setModalOpen(true)
+        }}>
+        {/* {props.button} */}
+        {props.btnStyle === 'star' ? <FontAwesomeIcon  style={second} icon={faStar} />: <FontAwesomeIcon  style={second} icon={faCircleXmark} />}
+        </RelatedBtn>
```
Comment (Nkozlare) [inline client/src/components/relatedProducts/card.jsx:129]: You left the console log in on line 124.

File: client/src/components/relatedProducts/relatedCarousel.jsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+             />
+          )
+        })}
+
+      </Carousel>
+    </div>
+  );
+}
+
+export default RelatedCarousel;
```
Comment (Nkozlare) [inline client/src/components/relatedProducts/relatedCarousel.jsx:106]: Is this the second carousel? This is a lot of code for one day.

**Review by Nkozlare — APPROVED** [review]
It looks like this was a lot of modularization. Something that I should do if I have a little bit of time left over. But it looks a lot cleaner now. You are missing a lot of your semicolons though.

File: client/src/components/relatedProducts/card.jsx (hunk 1 above)
Comment (Chrisgood2go) [reply client/src/components/relatedProducts/card.jsx:12]: good to know, thanks. will fix that.

File: client/src/components/relatedProducts/card.jsx (hunk 2 above)
Comment (Chrisgood2go) [reply client/src/components/relatedProducts/card.jsx:129]: Im keeping it for know, it only logs to console when clicked

File: client/src/components/relatedProducts/relatedCarousel.jsx (hunk 3 above)
Comment (Chrisgood2go) [reply client/src/components/relatedProducts/relatedCarousel.jsx:106]: a-lot of copying from where it originally was.

**Pull request closed without merging**